subprojects/dependency-management/src/main/java/org/gradle/api/internal/notations/DependencyNotationParser.java

```diff
@@ -79,24 +77,6 @@ public static DependencyNotationParser create(Instantiator instantiator,
             .converter(mapNotationConverter)
             .fromType(FileCollection.class, filesNotationConverter)
             .fromType(Project.class, projectNotationConverter)
-            .fromType(DependencyFactory.ClassPathNotation.class, new NotationConverter<DependencyFactory.ClassPathNotation, Dependency>() {
```

Comment:
I am not sure why this was removed. This is still needed to support the notation, right?

Reply:
Isn't line 100 enough?

Reply:
That's the new notation. Won't using the deprecated notation fail without this converter?

Reply:
I thought that was the idea, it's marked for removal in Gradle 8 below.